html/docs/include/export.php

```diff
@@ -802,5 +705,7 @@
 		structures to allow code coverage. This speeds up Xdebug quite a bit,
 		but of course, [FEAT:code_coverage] won't work.",
 		FUNC_CODE_COVERAGE
-	),
-);
+	)] as $i => $arr) {
+	    file_put_contents(__DIR__ . '/settings/' . $i . '.php', $arr[3]);
+	
+	}
```

Comment:
According to PSR-2 (which we mostly follow at Vimeo, also all PHP code I've seen in the last five years):

`The closing ?> tag MUST be omitted from files containing only PHP.`

Reply:
Yes, I know what PSR-2 says about it, but I still disagree :-þ

Reply:
ok! your project, your rules

Reply:
Do I dare to ask whether you've used spaces as well? ;-) I'm a tabs person (writing C is what does that to you!)

Reply:
I don’t like how tabs look in GH, but I’m happy to accommodate. I used tabs writing C#, but PHP community almost universally prefers spaces.

Reply:
Sorry, one more question about this because it's strange enough to worry me.

If a file ends in `?>` followed by a newline character, wouldn't the PHP engine try to output that character somewhere?

Reply:
No :-) The `?>` token eats the `\n`: https://github.com/php/php-src/blob/master/Zend/zend_language_scanner.l#L2197